### DISCUSSION TITLE
리팩터링 7주차 안건

### DISCUSSION THREAD

**Discussion by gitul0515** [prgrms-web-devcourse/Team-BackFro-ArtZip-FE#303] [02.Refactoring]
ReviewFeed에서 next/image를 사용하고는 있으나, 
이미지 용량에 대한 최적화는 아직 이루어지지 못한 것 같습니다. 
`Intrinsic size`를 `Rendered size`의 약 1~2배 정도로 줄이는 것이 필요합니다. 

![image](https://user-images.githubusercontent.com/80658269/192992102-5ec9f3cd-33e2-459e-a114-311fedbd2ab6.png)

**Comment by gitul0515** [comment]
Secure cookie

**Reply by punchdrunkard** [reply]
이 경우, 자바스크립트 접근을 막는다는 특성 때문에 프론트엔드에서도 쿠키를 사용할 수 없어서 사용을 못하게 되었습니다 ㅠㅠ

**Comment by gitul0515** [comment]
Error Boundary

**Reply by gitul0515** [reply]
SSR을 썼기 때문에 CLS가 나쁘지 않다. 
pre-render가 되므로

**Reply by gitul0515** [reply]
스켈레톤
